Dockerfile

```diff
@@ -0,0 +1,29 @@
+FROM jboss/base-jdk:8
+
+MAINTAINER Andrew Lee Rubinger (https://github.com/ALRubinger)
+
+# install wget
+USER root
+
+RUN yum install -y wget
+
+# get maven 3.3.9
+RUN wget --no-verbose -O /tmp/apache-maven-3.3.9.tar.gz http://archive.apache.org/dist/maven/maven-3/3.3.9/binaries/apache-maven-3.3.9-bin.tar.gz
```

Comment:
Please squash wget and other RUN instructions to one (like in https://github.com/jboss-dockerfiles/wildfly/blob/master/Dockerfile#L11) to make image smaller (in the way you did the image layers will still contain maven tarballs, etc.)